### ISSUE TITLE
Crashing app C++ Exception N8facebook3jsi7JSErrorE

### ISSUE THREAD

**Issue by redbluenat** [refiner-io/mobile-sdk-react-native#34]
I can see in Sentry that the refiner is crashing the app 

C++ Exception
Ongoing
Level: Fatal
Unhandled
N8facebook3jsi7JSErrorE

<img width="1092" alt="Screenshot 2024-10-31 at 11 15 11 am" src="https://github.com/user-attachments/assets/a7e80616-8879-48be-90dd-c9ea98cf63fc">

It is happening for iOS users only. I can't really reproduce it, but maybe somebody is facing this issue too?

**Comment by tekinalper** [comment]
@redbluenat 

1. What are your React Native and Refiner SDK versions? 
2. When did the crashes start according to the reports?
3. Did you recently change anything regarding the surveys?

**Comment by redbluenat** [comment]
@tekinalper React Native version 0.73.9 and Refiner version 1.3.12. The first crash appeared after updating React Native to version 0.73. I didn't make any changes to the surveys.

**Comment by tekinalper** [comment]
Do you still have this issue? We upgraded RN and updated other things so recommend to upgrade Refiner SDK version.

**Comment by redbluenat** [comment]
@tekinalper I tried to update, but I can't buildthe  iOS project:

<img width="257" alt="Image" src="https://github.com/user-attachments/assets/dc14483e-3278-4a75-bc3c-09ed72989a9a" />

**Comment by tekinalper** [comment]
@redbluenat Could you also try running the example project?

**Comment by redbluenat** [comment]
The example project was built successfully, but I see a red screen. However, we are using two different RN versions. 

![Image](https://github.com/user-attachments/assets/d69e797c-8f12-4737-a2b0-fd1b1dd50083)

**Comment by tekinalper** [comment]
@redbluenat You're right. Maybe you can check https://github.com/software-mansion/react-native-gesture-handler/issues/2749 for the issue.

For testing purposes, you may switch to [this commit](https://github.com/refiner-io/mobile-sdk-react-native/tree/056a1bd36ead8bc2e3af2c37ef7c4b47f382c5fc) and try to run the example project by upgrading Refiner version to 1.5.0 (Refiner 1.5.0 also uses RN 0.73.2).

**Comment by redbluenat** [comment]
@tekinalper I tried the version 1.5.0 and still have the same issue:

<img width="845" alt="Image" src="https://github.com/user-attachments/assets/11b54d2c-228c-4a19-ac2e-25e67476ee5a" />

**Comment by tekinalper** [comment]
@redbluenat you ran pod install command, right?

**Comment by tekinalper** [comment]
@redbluenat Can you reach me via email to organize a call to review the issues? I can't reproduce the case in my environment.

**Comment by tekinalper** [comment]
It was pod file update issue, 1.5.0 works.
